hyperspy/_components/gaussian2d.py

```diff
@@ -90,3 +109,12 @@ def fwhm_y(self):
     @fwhm_y.setter
     def fwhm_y(self, value):
         self.sigma_y.value = value / sigma2fwhm
+
+    # Rotation compared to "x"-axis
+    @property
+    def rotation_degrees(self):
+        """ Finds the rotation compared to the "x-axis" """
```

Comment:
The in-built .rotation parameter will already output the rotation in radians in relation to the x-axis. So having a convenience function which outputs the value in degrees is useful.